spec/BowlSpec.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+describe('Bowl', function() {
+
+  var bowl;
+  // similar to an instance variable initialization.
+
+  beforeEach(function() {
+    bowl = new Bowl(0,0,0,0);
+    // creates a new object before each test.
+  });
+
+  it('The new bowl has a Score which starts at 0', function() {
```

Comment:
good for setting up, however, this test is checking the **state** (attribute) rather than **behaviour**.

testing behaviour: setting up a function called **'currentScore()'** which calls the attribute _score_ . Hope it makes sense!